### DISCUSSION TITLE
Debug renderer no longer seems to work

### DISCUSSION THREAD

**Discussion by profbudinga** [DanielChappuis/reactphysics3d#374] [Q&A] [answered]
Hi Guys,

I just updated my code to work with the latest version 0.10, and now the debug renderer does not generate any triangles for me to render, the collisions are still working correctly , but i am thinking i must be missing something that has been changed, anybody else having the same issue?

Thanks

**Comment by DanielChappuis** [comment, marked as answer]
Hello. It's probably because of a change in the API.

As you can read in the [change log](https://github.com/DanielChappuis/reactphysics3d/releases/tag/v0.10.0) of the release, 

> It is now necessary to enable debug rendering for each body that you want to debug using the Body::setIsDebugEnabled() method

Now, by default debug renderer is disabled for each body and you need enable it for the bodies you want to debug.

**Reply by profbudinga** [reply]
Thank you that is exactly what I was looking for.
